src/main/java/menion/android/whereyougo/gui/dialog/NoPasswordDialogFragment.java

```diff
@@ -0,0 +1,52 @@
+package menion.android.whereyougo.gui.dialog;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.os.Bundle;
+
+import androidx.fragment.app.DialogFragment;
+
+import menion.android.whereyougo.R;
+
+public class NoPasswordDialogFragment extends DialogFragment {
+
+    private NoPasswordDialogListener listener;
+
+    public interface NoPasswordDialogListener {
+        public void onPositiveClick(DialogFragment dialog);
+    }
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+
+        try {
+            listener = (NoPasswordDialogListener) context;
+        } catch (ClassCastException e) {
+            throw new ClassCastException(getActivity().toString() + "must implement NoPasswordDialogListener.");
+        }
+    }
+
+    @Override
+    public Dialog onCreateDialog(Bundle savedInstanceState) {
+        // Use the Builder class for convenient dialog construction
+        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
+        builder.setMessage(R.string.dialog_no_password)
+                .setPositiveButton(R.string.ok, new DialogInterface.OnClickListener() {
+                    public void onClick(DialogInterface dialog, int id) {
+                        listener.onPositiveClick(NoPasswordDialogFragment.this);
+                    }
+                })
+                .setNegativeButton(R.string.cancel, new DialogInterface.OnClickListener() {
+                    public void onClick(DialogInterface dialog, int id) {
+                        // User cancelled the dialog --> Do nothing
+                    }
+                });
```

Comment:
From AndroidStudio:
Anonymous type can be replaced by lambda
~~~
                .setNegativeButton(R.string.cancel, (dialog, id) -> {
                    // User cancelled the dialog --> Do nothing
                });
~~~

Reply:
Fixed by c1bf2e3